policies/project-director-election-process.md

```diff
@@ -0,0 +1,77 @@
+# Project Director Election Process
+
+## Goals
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria in the [role description document](TODO: link), focusing on new leadership and diverse perspectives rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity by prioritizing selecting candidates that everyone can accept over selecting a subset of people's first choice.
+* Conduct the selection process in a time-efficient manner.
+* Incorporate input from a wide range of stakeholders across the entire Rust project.
+* Ensure a diverse pool of qualified candidates for selection.
+* Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than preference or criticism.
+* Ensure participants feel positive about their interactions and the execution of the process.
+* Ensure the process is compliant with Rust Foundation bylaws and by extension Delaware corporation law.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone the council trusts to set aside personal preferences during the election process.
+    * **Goals of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944)). The role description must contain the following content at a minimum:
+    * Goals
+    * Time Investment
+    * Responsibilities
+    * Qualifications
+    * Term
+
+### Pre-Meeting Coordination
+
+* Gather nominations
+    * The nomination process begins with an announcement from the facilitator chosen in the Setup phase. Council representatives should help publicize this announcement to their teams as well.
```

Comment:
I feel like that's implied in the way this is already worded. Do you have a suggestion for how to make it more explicit?

Reply:
I think this would resolve my nit:

```suggestion
    * The nomination process begins with a public announcement made on public communication channels from the facilitator chosen in the Setup phase. Council representatives should help publicize this announcement to their teams as well.
```

Reply:
Does a public Zulip stream count as a public communication channel?